docs/ibc/3_IBC_TERMINOLOGY.md

```diff
@@ -38,9 +38,9 @@ A *header* is an update to the consensus state of a particular blockchain that c
 
 *Finality* is the guarantee provided by a consensus algorithm that a particular block will not be reverted, subject to certain assumptions about the behavior of the validator set. The IBC protocol requires finality.
 
-### Accumulator
+### Commitment 
 
-A cryptographic *accumulator* is a way to cheaply verify membership of a value in a set, where the set can be commited to with a short witness string. An *accumulator proof* refers to the proof structure which proves whether a particular value is or is not a member of a particular committed-to set.
+A cryptographic *commitment* is a way to cheaply verify membership of a key => value pair in a mapping, where the mapping can be commited to with a short witness string. An *vector commitment proof* refers to the proof structure which proves whether a particular key maps to a particular value in a committed-to set or not.
```

Comment:
Perhaps reword to state that we are using `commitment` as an alias for `vector commitment`.  Colloquially this is right, though precisely a cryptographic commitment is something akin to a hash and reveal scheme. (You commit to a value in a way that it is computationally binding, and then can reveal the bound value later)

https://en.wikipedia.org/wiki/Commitment_scheme